assets/stylesheets/modules/ai-helper/common/ai-helper.scss

```diff
@@ -292,17 +292,13 @@
   position: relative;
 }
 
-// Prevent suggestion button from wrapping on smaller screens
-@media screen and (max-width: 768px) {
-  #reply-control .composer-fields.showing-ai-suggestions {
-    .category-input {
-      .category-chooser {
-        width: 10px;
-      }
-
-      + .mini-tag-chooser {
-        width: 10px;
-      }
+// Prevent suggestion button from wrapping
+#reply-control {
+  .with-category .showing-ai-suggestions .category-input,
+  .with-tags .showing-ai-suggestions .mini-tag-chooser {
+    flex-wrap: nowrap;
+    .select-kit {
+      max-width: calc(100% - 34px);
```

Comment:
This bit covers screen size of about 770px to 1200px.

Without:

![CleanShot 2023-11-03 at 19 48 39](https://github.com/discourse/discourse-ai/assets/69276978/6ba504e6-2240-4e14-aa01-ed5043234de6)

With:

![CleanShot 2023-11-03 at 19 49 18](https://github.com/discourse/discourse-ai/assets/69276978/fd94ac93-1e5e-46a6-a109-f79e238cd53a)